src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -137,238 +139,291 @@ public void AndroidStudioConverter_ConvertToSarifIssue_HintsAreStapledToDescript
             };
 
             var uut = new AndroidStudioProblem(builder);
-            Result result = AndroidStudioConverter.ConvertProblemToSarifIssue(uut);
+            Result result = new AndroidStudioConverter().ConvertProblemToSarifResult(uut);
```

Comment:
`ConvertProblemToSarifResult` is now an instance method. The reason is that, as we process each result, we must save the logical location information that we'll write out to the log later. We could have made the property in which we save that information (`LogicalLocationsDictionary`, which you'll see later) a static property, but it didn't feel right.